tests/test_complete_rules.py

```diff
@@ -936,11 +967,38 @@ def match(self, node: ast.AST) -> Iterator[Replace | InsertAfter]:
             new_trys.append(new_try)
 
         first_try, *remaining_trys = new_trys
+        print(ast.unparse(node))
+        print(ast.unparse(first_try))
         yield Replace(node, first_try)
         for remaining_try in reversed(remaining_trys):
             yield InsertAfter(node, remaining_try)
 
 
+class WrapInMultilineFstring(Rule):
+    INPUT_SOURCE = '''
+def f():
+    return """
+a
+"""
+'''
+    EXPECTED_SOURCE = '''
+def f():
+    return F("""
+a
```

Comment:
Nice!!! So glad this PR also solves this issue as well <3

Reply:
Yes, but as an patch on a patch, I felt it is clunky, but if you approve it then yes